### PR TITLE
[dashboard] remove symbolic link

### PR DESCRIPTION
**Pull request by aslonnie**
move into ray package, remove the one from root.

### CODE REVIEW

**Comment by jjyao** [comment]
@pcmoritz may have context why we had symlink in the first place and whether we want to remove it

**Comment by aslonnie** [comment]
past comments from @ericl :

- eric: it was originally to make rllib more clickable.
- eric: it sounds fine to reverse the direction now; this was something we tried a long time ago.
- sang: dashboard just follows what rllib did
- koroush: we can totally swap the symlink direction.
- sven: fine with me, too, for RLlib.
- ed: yep ant just copied what rllib did for dashboard


I am fixing dashboard first, and then rllib. rllib is a bit tricky to fix as there are some links in the docs that point to rllib dir.

**Comment by aslonnie** [comment]
this will make it easier / simpler to build ray on windows and for conda-forge.

File: python/ray/dashboard/modules/actor/actor_head.py [hunk 1]
```diff
@@ -2,19 +2,15 @@
...
 
 import ray.dashboard.optional_utils as dashboard_optional_utils
 import ray.dashboard.utils as dashboard_utils
 from ray._private.gcs_pubsub import GcsAioActorSubscriber
-from ray.core.generated import (
-    gcs_service_pb2,
-    gcs_service_pb2_grpc,
-)
-from ray.dashboard.datacenter import DataSource, DataOrganizer
+from ray.core.generated import gcs_service_pb2, gcs_service_pb2_grpc
```
Comment (alanwguo) [inline python/ray/dashboard/modules/actor/actor_head.py:12]: are these import changes auto-fixed by a linter or tool?

**Review by alanwguo — COMMENTED** [review]
high level, I have no concerns with the change.

Although the PR does have a lot of import ordering changes that seem unrelated making it harder to review.

File: python/ray/dashboard/modules/actor/actor_head.py (hunk 1 above)
Comment (aslonnie) [reply python/ray/dashboard/modules/actor/actor_head.py:12]: yes.. let me try to split these into a separate PR.

`dashboard/` dir was previously not included in the formatter..

Comment (aslonnie) [reply python/ray/dashboard/modules/actor/actor_head.py:12]: https://github.com/ray-project/ray/pull/46483 PR applying isort without moving the files.

we can approve that first.

Comment (aslonnie) [reply python/ray/dashboard/modules/actor/actor_head.py]: updated, now `dashboard/` files are pure location moving.

**Review by alanwguo — APPROVED** [review]

**Review by can-anyscale — APPROVED** [review]
approve for doc changes

**Comment by aslonnie** [comment]
merging with @alanwguo 's blessings.

**Comment by can-anyscale** [comment]
window wheels seem to fail starting on this commit https://buildkite.com/ray-project/postmerge/builds/5344

**Comment by aslonnie** [comment]
> window wheels seem to fail starting on this commit https://buildkite.com/ray-project/postmerge/builds/5344

hmm... looking at the logs the npm building part already succeeded, then the pip install thirdparty vendored files failed..

**Pull request merged**